src/Owin.Security.Providers.Podbean/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,19 @@
+﻿#region
```

Comment:
Please do not wrap using statements in regions

Reply:
I removed the region.